### DISCUSSION TITLE
Support Request: Windows Ink not working (Cannot find VirtualHID. Install VMulti driver)

### DISCUSSION THREAD

**Discussion by lbx42** [OpenTabletDriver/OpenTabletDriver#3272] [General]
### What do you need help with?

Whenever I have my Output set to Windows Ink Absolute my pen doesn't work, it only works when its on regular Absolute. I'm not sure why this is happening considering I've been using it for a while and it was fine. I've looked for fixes in the GitHub thread and also in the Discord but nothing has helped.

### What have you tried?

I've tried uninstalling VMulti Driver multiple times, restarting my pc, reinstalling OpenTabletDriver, using TabletDriverCleanup and uninstalling everything and reinstalling it, checking Powershell to see if VMulti is installed and it is

### Diagnostic Information

[OTD 1.json](https://github.com/OpenTabletDriver/OpenTabletDriver/files/14836587/OTD.1.json)
[OTD 1 - .txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/14836610/OTD.1.-.txt)

### Acknowledgements

- [X] I have installed OpenTabletDriver by following its [official installation instructions](https://opentabletdriver.net/).
- [x] I have checked the [OpenTabletDriver Wiki](https://opentabletdriver.net/Wiki) and respective FAQ pages, and my issue was not covered or actually fixed.
- [X] I have searched the existing issues and this new issue is not a duplicate of any.
- [X] I have written a concise and meaningful title.
- [X] I am on the latest version of OpenTabletDriver.

**Comment by jamesbt365** [comment]
Remove the entire %localappdata%\OpenTabletDriver directory with the driver closed, then follow the [plugins wiki](https://github.com/X9VoiD/VoiDPlugins/wiki/WindowsInk) again.

**Reply by lbx42** [reply]
I tried doing that but it's still giving me the same error

**Reply by jamesbt365** [reply]
export diagnostic again.

**Reply by lbx42** [reply]
[OTD 2.json](https://github.com/OpenTabletDriver/OpenTabletDriver/files/14836795/OTD.2.json)

**Reply by jamesbt365** [reply]
Run TabletDriverCleanup, uninstall everything it asks about and then go into device manager > view > device by container, screenshot he whole list.

**Reply by lbx42** [reply]
![image](https://github.com/OpenTabletDriver/OpenTabletDriver/assets/157870766/83517c6d-32a9-4e6c-96ee-f9a2778cc0f5)

**Reply by lbx42** [reply]
![image](https://github.com/OpenTabletDriver/OpenTabletDriver/assets/157870766/0062273a-c369-4b6f-b3f5-f41c7b0483f7)

**Reply by jamesbt365** [reply]
If you ran the vmulti installation script and vmulti driver did not install then there isn't anything we can do, this is a problem with your system that you will have to debug yourself. VMulti is not developed by us so there isn't anything we can do to help here.
